js/os.ts

```diff
@@ -130,3 +130,22 @@ export function start(source?: string): msg.StartRes {
 
   return startResMsg;
 }
+
+/**
+ * Returns the current user's home directory.
+ */
+export function userHomeDir(): string {
```

Comment:
Can we call it `homeDir` instead of `userHomeDir` ?

Note in the jsdoc that this does not require elevated privileges.